content/documentation/pages/4-batch-developer-guides/2-batch/5-data-flow-composed-task.md

```diff
@@ -6,36 +6,105 @@ description: 'Create and launch a Composed Task using Data Flow'
 
 # Create and launch a Composed Task using Data Flow
 
-In this section we introduce composed tasks, show how to register task applications with Data Flow, create a composed task DSL, and launch the composed task to Cloud Foundry, Kubernetes and your local machine.
+In our previous sections we used two applications:
 
-## Development
+- billsetuptask: The billsetuptask setups up the database for the billrun application.
+- billrun: The billrun reads usage information from a file and generates a billing statement record for each entry from a usage file.
 
-Explaine what we are doing to do development wise.
+Previously we executed the billsetuptask app and then the billrun app in sequence manually.
+This certainly works for the small case where we want to run 2 apps back to back where the first app runs in a few seconds.
+But what about the cases where there are 5, 10, 20 or even more apps that need to be run in a sequence and the runtime of some the apps maybe hours.  
+Or cases where one of the apps returns a non zero exit code. What do we do in this case?
+Spring Cloud Data Flow provides a solution for this issue through the use of Composed Tasks.  
+A composed task is a directed graph where each node of the graph is a task application.
+To use the case above a graph would look something like this:
+![Bill Run Composed Task](images/SCDF-composed-task-simple.png)
 
-### Application Registration
+In this section we show how to create a composed task using Spring Cloud Data Flow's UI, and then launch the composed task to Cloud Foundry, Kubernetes and your local machine.
 
-how to register apps using the CLI and/or (?) UI
+## Prerequisites
 
-### Create Stream DSL
+### Data Flow Installation
 
-how to create the DSL using CLI and/or (?) UI
+Make sure you have installed Spring Cloud Data Flow to the platform of your choice:
 
-### Validate Stream DSL
+- [Local](%currentPath%/installation/local/)
+- [Cloud Foundry](%currentPath%/installation/cloudfoundry)
+- [Kubernetes](%currentPath%/installation/kubernetes/)
 
-make sure the app coordinates are correct
+### Register the Billing Applications
 
-## Deployment
+For this guide, we will need for you to have registered the [billsetuptask](%currentPath%/batch-developer-guides/batch/data-flow-simple-task) and the [billrun](%currentPath%/batch-developer-guides/batch/data-flow-spring-batch) applications in Spring Cloud Data Flow.
 
-Deploy to local, Cloud Foundry and Kubernetes
+### Register the Composed Task Runner
 
-### Local
+If you haven't already done so import the [Spring Cloud Task App Starters](https://cloud.spring.io/spring-cloud-task-app-starters/), which will give us the `composed-task-runner` application.
```

Comment:
Add comma?? : If you haven't already done so import --> If you haven't already done so, import